examples/cmip_clims_param_nocmor.py

```diff
@@ -0,0 +1,23 @@
+
+
+vars = ['pr','rlut', 'rsut','rsutcs','rlutcs','tas','prw','tauu','tauv','uas','vas','psl','hus','ta','ua','va', 'zg']
+vars = ['rlut']
+
+realm = 'atm'
+MIP = 'cmip5'
+exp = 'historical'
+
+modpath = '/work/cmip5-test/' + exp + '/' + realm + '/mo_new/'
```

Comment:
@doutriaux1 do we want to change this to `os.path.join()`?

Reply:
@durack1 actually the _nocmor stuff needs to go away